### ISSUE TITLE
add pronunciation for raw Unicode braille cells

### ISSUE THREAD

**Issue by saizai** [nvaccess/nvda#13778]
If Unicode braille occurs within text — for instance, ⠐⠣⠃⠗⠇⠐⠜ says "(braille)" in UEB — NVDA currently just skips over the braille entirely. It should at least pronounce the braille cells.

Because this is pronunciation for the Unicode characters themselves, it should be braille-code-agnostic and just pronounce individual braille characters in the usual "dot 3 5 7" way.

I've made a pull request that does this: https://github.com/nvaccess/nvda/pull/13777

Separately — my pull request does not address this — NVDA should be able to interpret Unicode braille that's embedded in non-braille text based on its current input/output code table settings, and just bracket that by some sort of indicator. E.g. if the output is UEB2 braille, use ⠐⠷⠑⠝⠤⠥⠑⠃⠼⠃⠄… ⠠⠐⠾  to indicate a Unicode-braille UEB grade 2 passage within auto-translated text, or ⠐⠷⠃⠗⠇⠄… ⠠⠐⠾ to indicate Unicode braille of unknown code; if the output is English speech, have the narrator speak "begin Unicode braille" and "end Unicode braille" to bracket it.

So e.g.:

* text being presented: `for instance, ⠐⠣⠃⠗⠇⠐⠜ says "(braille)" in UEB`
* UEB2 braille display output (modify as appropriate for other output codes): `⠿⠀⠔⠌⠨⠑⠂⠀⠐⠷⠃⠗⠇⠠⠀⠐⠣⠃⠗⠇⠐⠜⠀⠠⠐⠾⠀⠎⠁⠽⠎⠀⠦⠐⠣⠃⠖⠇⠐⠜⠴⠀⠔⠀⠠⠠⠥⠑⠃`
  * Current NVDA UEB2 output: `⠿⠀⠔⠌⠨⠑⠂⠀⠐⠣⠃⠗⠇⠐⠜⠀⠎⠁⠽⠎⠀⠦⠐⠣⠃⠖⠇⠐⠜⠴⠀⠔⠀⠠⠠⠥⠑⠃`. This doesn't indicate the switch between NVDA-converted text and literal braille.
* English speech output (again, modify as appropriate for the quote and the localised way to speak a braille cell): `for instance, begin Unicode braille, dot 5 dot 1 2 6 dot 1 2 dot 1 2 3 5 dot 1 2 3 dot 5 dot 3 4 5, end Unicode braille, says quote left paren braille right paren quote in cap u e b`
  * better speech output: `for instance, begin Unicode braille, dots 5, 1 2 6, 1 2, 1 2 3 5, 1 2 3, 5, 3 4 5, end Unicode braille, says quote left paren braille right paren quote in cap u e b`. 
  * current NVDA English output (in my config): `for instance comma says left paren, braille, right paren, in u e b`. This totally omits the Unicode braille, and doesn't even indicate that anything is there.
 
The better version would require some sort of mode flag in the pronunciation code to indicate that 'dot' doesn't need to be repeated in contiguous Unicode braille. I don't know how to do that in NVDA, and have not attempted to do so here. This PR does at least the minimum possible thing to make Unicode braille audible at all, which is an improvement over the current status.

I would also suggest that if NVDA encounters something it doesn't know how to pronounce, its speech output should at the very least give some indication that _something_ was there, like with a special beep. (This is as opposed to encountering punctuation it knows how to pronounce and is _set to ignore_, like the quote marks and all-caps in the above under my config. I did not tell it to ignore the character ⠿, but if I rely on my speech output, it's completely invisible.)
